src/test/datascience/notebook/executionService.vscode.test.ts

```diff
@@ -54,18 +54,23 @@ suite('DataScience - VSCode Notebook - (Execution) (slow)', () => {
         editorProvider = api.serviceContainer.get<INotebookEditorProvider>(INotebookEditorProvider);
     });
     // Use same notebook without starting kernel in every single test (use one for whole suite).
-    setup(async () => {
+    setup(async function () {
+        traceInfo(`Start Test ${this.currentTest?.title}`);
         sinon.restore();
         // Open a notebook and use this for all tests in this test suite.
         await editorProvider.createNew();
         await waitForKernelToGetAutoSelected();
         await deleteAllCellsAndWait();
         assert.isOk(vscodeNotebook.activeNotebookEditor, 'No active notebook');
+        traceInfo(`Start Test (completed) ${this.currentTest?.title}`);
     });
-    teardown(async () => {
+    teardown(async function () {
+        traceInfo(`Ended Test ${this.currentTest?.title}`);
         // Added temporarily to identify why tests are failing.
         process.env.VSC_JUPYTER_LOG_KERNEL_OUTPUT = undefined;
         await closeNotebooks(disposables);
+        await closeNotebooksAndCleanUpAfterTests(disposables);
+        traceInfo(`Ended Test (completed) ${this.currentTest?.title}`);
```

Comment:
Add logs to see when a test starts & ends.
This is to identify what's going on when a test starts, I want to ensure we dont have code/processes from other (old tests) interfering with the current test.

Noce that I have added `closeNotebooksAndCleanUpAfterTest` this might not be necessary.
That method ensures we shutdown all INoteook objects for each test.

Previously i found that when running a test, we're still getting output from previous test & that was confusing & getting in the way of identifying why a test was failing.

Once we have resolved flakyness, i'll remove thse `Start Test,, End Test` messages.